README.md

```diff
@@ -1,40 +1,51 @@
 # Calhoun
-Notebook preview service
+Jupyter notebook preview service
 
-### Requirements
-- Docker
-- Node 8
-- Google cloud SDK
+### Background
+This project is part of the Terra (né FireCloud) platform of service APIs. It is primarily intended to provide preview versions of notebooks for Terra-based web applications, such as the [Terra workbench](https://www.terra.bio) research environment. It is based on [Calhoun](https://github.com/DataBiosphere/calhoun), a similar service written in Node.js. The service is written wholly in Python 3.7, so that it can be deployed in the Google App Engine Standard environment. Calhoun used Node and shelled out to nbconvert. Since it required both Node and Python, it needed GAE Flexible which lacks FedRAMP certification.
```

Comment:
"It is based on [Calhoun]" is a weird sentence since we're still Calhoun.  maybe just "it was previously written in Node.js

Reply:
Yup. This was in a separate repo before deciding to make a big shift in this existing repo. I'll revise.